packages/agent/package.json

```diff
@@ -35,7 +35,8 @@
     "playwright": "^0.11.1",
     "regenerator-runtime": "^0.13.3",
     "ts-node": "*",
-    "typescript": "^3.7.0"
+    "typescript": "^3.7.0",
+    "@bigtest/effection": "*"
```

Comment:
`@bigtest/effection` is a runtime dependency

Reply:
Done